grpc-gcp/src/main/java/com/google/grpc/gcp/GcpManagedChannel.java

```diff
@@ -555,27 +654,27 @@ protected int getId() {
     }
 
     protected void affinityCountIncr() {
-      affinityCount++;
+      affinityCount.incrementAndGet();
```

Comment:
This method and and the next 5 method always are called from synchronized blocks.
they can safely be ints.
please verify..

Reply:
Moved to another PR.